src/frontend/screens/Observation/Buttons.tsx

```diff
@@ -162,11 +162,13 @@ export const ButtonFields = ({
         completedFields.push({label: field.label, value: displayedValue});
       }
 
+      console.log({length: base64Urls.length});
+
       await Share.open({
         subject: `${t(m.comapeoAlert)} — _*${preset ? preset.name : t(m.fallbackCategoryName)}*_ — ${formatDate(observation.createdAt, {format: 'long'})}`,
         title:
           base64Urls.length > 0 ? t(m.shareMediaTitle) : t(m.shareTextTitle),
-        urls: base64Urls,
+        urls: !base64Urls.length ? undefined : base64Urls,
```

Comment:
is there any need to avoid passing in an empty array? would think what was here before should be okay

otherwise, a suggestion the the ternary

```suggestion
        urls: base64Urls.length ? base64Urls : undefind,
```

Reply:
passing the empty array was breaking it in the newer version (not in the changelog).